draft-ietf-quic-qpack.md

```diff
@@ -324,10 +332,9 @@ permit the encoder to track the decoder's state.  These events are:
 
 Knowledge that a header block with references to the dynamic table has been
 processed permits the encoder to evict entries to which no unacknowledged
-references remain, regardless of whether those references were potentially
-blocking (see {{blocked-insertion}}).  When a stream is reset or abandoned, the
-indication that these header blocks will never be processed serves a similar
-function; see {{stream-cancellation}}.
+references remain; see {{blocked-insertion}}.  When a stream is reset or
```

Comment:
Style nit: I think that we've been using the parenthetical form for this sort of reference.  I think that this is right, even the semi-colon, which I initially stumbled over, but consistency is somewhat important.

Reply:
I agree with you.  Sorry, I got inspired by the initial semicolon in this section.